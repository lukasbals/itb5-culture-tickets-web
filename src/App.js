import './App.css';
import { Button } from 'antd';
import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
  text-align: center;
`;

function App() {
  return (
    <Container>
      <h1>Culture Tickets Web Console</h1>
      <Button>Ant Design Button</Button>
    </Container>
  );
}

export default App;
