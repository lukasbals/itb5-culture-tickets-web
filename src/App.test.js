import { render } from '@testing-library/react';
import App from './App';
import React from 'react';

test('App should render correctly', () => {
  const { baseElement } = render(<App/>);
  expect(baseElement).toMatchSnapshot();
});
