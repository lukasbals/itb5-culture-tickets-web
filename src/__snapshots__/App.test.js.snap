// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<body>
  <div>
    <div
      class="sc-bdVaJa khUSez"
    >
      <h1>
        Culture Tickets Web Console
      </h1>
      <button
        class="ant-btn"
        type="button"
      >
        <span>
          Ant Design Button
        </span>
      </button>
    </div>
  </div>
</body>
`;
